src/test/java/org/xstefank/ci/TestCI.java

```diff
@@ -0,0 +1,31 @@
+package org.xstefank.ci;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+public class TestCI implements ContinuousIntegration {
+
+    private static final String NAME = "TestCI";
+    private boolean triggered;
+
+    public static TestCI getTestCIFromLoader() {
+        return (TestCI) CILoader.getCI(NAME);
+    }
+
+    public static void putTestCIToLoader() {
+        CILoader.addCI(NAME, new TestCI());
+    }
```

Comment:
Well thanks for the question.. I think I need just this method because the method "addCI" is package private and I can't call it from "CommandsTest" class. Maybe I can try to refactor Commands to different package if it does make sense?